hooks/image/60-kolibri-content

```diff
@@ -50,9 +51,16 @@ cat << EOF > "${KOLIBRI_HOME}"/options.ini
 [Deployment]
 STATIC_USE_SYMLINKS = 0
 EOF
+if [ "${EIB_KOLIBRI_DESKTOP_AUTH_PLUGIN_REGULAR_USERS_CAN_MANAGE_CONTENT}" = "true" ]; then
+    cat << EOF >> "${KOLIBRI_HOME}"/options.ini
+[DesktopAuth]
+REGULAR_USERS_CAN_MANAGE_CONTENT = True
+EOF
+fi
```

Comment:
This hook is only going to run if `EIB_KOLIBRI_INSTALL_CHANNELS` is set, which is not the case for all of the images where we expect `EIB_KOLIBRI_DESKTOP_AUTH_PLUGIN_REGULAR_USERS_CAN_MANAGE_CONTENT` to be true. Instead, I think we should do this in its own hook, similar to #55.